assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -0,0 +1,22 @@
+.wc-block-cart__sidebar {
+	border: 1px solid $core-grey-light-600;
+	border-width: 1px 0;
+}
```

Comment:
Another place where the sidebar feels like a `<Card/>` or `<Panel/>` type component.